README.md

```diff
@@ -22,7 +26,7 @@ This configuration limits memory usage to 512mb. This should be enough for most
 - "ES_JAVA_OPTS=-Xms512m -Xmx512m"` environment variable in `docker-compose.elasticsearch.yaml`
 ```
 
-If you change this variable, make sure to remove the `#ddev-generated` line at the top of the file. 
+If you change this variable, make sure to remove the `#ddev-generated` line at the top of the file.
```

Comment:
Change this in an extra docker-compose.elasticseearch-extras.yaml or whatever.

Reply:
Do you mean the removed trailing whitespace? That was just because of auto formatting.

Reply:
Now that you use a separate file, the changing of `#ddev-generated` is not necessary. You're no longer needing to edit the file provided by the add-on. 

```suggestion
```

Reply:
Well, that line is not provided by the PR. It's just the removed trailing whitespace at the end of the line:

<img width="328" alt="image" src="https://github.com/ddev/ddev-elasticsearch/assets/7961978/3c468b77-153d-4993-9f06-14277064a701">

Which was done by my auto markdown formatting during save. I can insert the whitespace because it is not a related change. Just some cleanup.

Reply:
I changed it back to how it was before. So, no need to discuss anymore. 😊